MLOps/examples/NYCTaxi/Taxi Datasets/lookup-ipyheader.csv

```diff
@@ -0,0 +1,264 @@
+pqlookup.long,pqlookup.lat,pqlookup.borough,pqlookup.location_i,pqlookup.objectid,pqlookup.shape_area,pqlookup.shape_leng,pqlookup.zone
```

Comment:
In your readme, please describe the data also

Reply:
I have just added a description of the data.